Makefile

```
@@ -262,6 +290,46 @@ else
 	$(error addlicense must be installed for this rule: go get -u github.com/google/addlicense)
 endif
 
+### gen_csv: generate the csv for a newer version
+gen_csv: update_devworkspace_crds update_terminal_manifests
+	operator-sdk generate csv --apis-dir ./pkg/apis --csv-version 1.0.0 --make-manifests --update-crds --operator-name "web-terminal" --output-dir ./web-terminal-operator
+	
+	# filter the deployments so that only the valid deployment is available. See: https://github.com/eclipse/che/issues/17010
+	cat ./web-terminal-operator/manifests/web-terminal.clusterserviceversion.yaml | \
+	yq -Y \
+	'.spec.install.spec.deployments[] |= select( .spec.selector.matchLabels.app? and .spec.selector.matchLabels.app=="che-workspace-controller")' | \
+	tee ./web-terminal-operator/manifests/web\ terminal.clusterserviceversion.yaml >>/dev/null
+
+	cp devworkspace-crds/deploy/crds/workspace.devfile.io_devworkspaces_crd.yaml web-terminal-operator/manifests
+
+### olm_build_bundle: build the bundle that will be stored on quay
+olm_build_bundle: _print_vars
+	# Create the bundle and push it to quay
+	operator-sdk bundle create $(BUNDLE_IMG) --channels alpha --package web-terminal --directory web-terminal-operator/manifests --overwrite --output-dir generated
+	docker push $(BUNDLE_IMG)
+
+#### olm_create_index: to create / update and push an index that contains the bundle
+olm_create_index:
+	opm index add -c docker --bundles $(BUNDLE_IMG) --tag $(INDEX_IMG)
+	docker push $(INDEX_IMG)
+
+### olm_start_local: use the catalogsource to make the operator be available on the marketplace. Must have $(INDEX_IMG) available on quay already and have it set to public
+olm_start_local: _print_vars
+	# replace references of catalogsource img with your image
+	sed -i.bak -e  "s|quay.io/che-incubator/che-workspace-operator-index:latest|$(INDEX_IMG)|g" ./deploy/olm-catalogsource/catalog-source.yaml
+	oc apply -f ./deploy/olm-catalogsource/catalog-source.yaml
+	sed -i.bak -e "s|$(INDEX_IMG)|quay.io/che-incubator/che-workspace-operator-index:latest|g" ./deploy/olm-catalogsource/catalog-source.yaml
+
+	# remove the .bak files
+	rm ./deploy/olm-catalogsource/catalog-source.yaml.bak
+
+### olm_full_start: build the catalog and deploys the catalog to the cluster without pushing olm files to application registry
+olm_full_start_local: _print_vars olm_build_bundle olm_create_index olm_start_local
```

```suggestion
olm_build_start_local: _print_vars olm_build_bundle olm_create_index olm_start_local
```
?

And it's terminal OLM... Then maybe even `terminal_olm_build_start_local` or `olm_terminal_build_start_local`? I know pretty long but it could help us avoid confusion later.